js/deepwaters.js

```diff
@@ -0,0 +1,1046 @@
+            'name': 'Deepwaters',
+            'countries': [ 'US' ],
+            'rateLimit': 5,
+            'certified': false,
+            'version': 'v1',
+            'has': {
+                'CORS': undefined,
+                'spot': true,
+                'margin': false,
+                'swap': false,
+                'future': false,
+                'option': false,
+                'cancelAllOrders': true,
+                'cancelOrder': true,
+                'createOrder': true,
+                'createStopLimitOrder': false,
+                'createStopMarketOrder': false,
+                'createStopOrder': false,
+                'fetchBalance': true,
+                'fetchBidsAsks': false,
+                'fetchBorrowRate': false,
+                'fetchBorrowRateHistories': false,
+                'fetchBorrowRateHistory': false,
+                'fetchBorrowRates': false,
+                'fetchBorrowRatesPerSymbol': false,
+                'fetchCanceledOrders': true,
+                'fetchClosedOrders': true,
+                'fetchCurrencies': true,
+                'fetchDepositAddress': false,
+                'fetchDeposits': false,
+                'fetchMarginMode': false,
+                'fetchMarkets': true,
+                'fetchMyTrades': true,
+                'fetchOHLCV': false,
+                'fetchOpenOrders': true,
+                'fetchOrder': true,
+                'fetchOrderBook': true,
+                'fetchOrders': true,
+                'fetchPositionMode': false,
+                'fetchStatus': false,
+                'fetchTicker': false,
+                'fetchTickers': false,
+                'fetchTime': false,
+                'fetchTrades': false,
+                'fetchTradingFee': false,
+                'fetchTradingFees': false,
+                'fetchTransactionFees': false,
+                'fetchTransactions': false,
+                'fetchTransfers': false,
+                'fetchWithdrawals': false,
+                'transfer': false,
+                'withdraw': false,
+            },
+            'timeframes': undefined,
+            'urls': {
+                'test': {
+                    'public': 'https://testnet.api.deepwaters.xyz/rest/v1',
+                    'private': 'https://testnet.api.deepwaters.xyz/rest/v1',
+                },
+                'api': {
+                    'public': 'https://api.deepwaters.xyz/rest/v1',
+                    'private': 'https://api.deepwaters.xyz/rest/v1',
+                },
+                'www': 'https://deepwaters.xyz',
+                'referral': '',
+                'doc': [
+                    'https://rest.docs.api.deepwaters.xyz/',
+                ],
+            },
+            'fees': {
+                'trading': {
+                    'tierBased': false,
+                    'percentage': true,
+                    'maker': 0.001,
+                    'taker': 0.0015,
+                    'feeSide': 'get',
```

Comment:
Could you find a source for the feeSide anywhere in their docs?

Reply:
Ben tamam sen